qbx_consumables/server.lua

```diff
@@ -120,6 +143,54 @@ lib.callback.register('consumables:server:usedItem', function(source, item)
 end)
 
 -- Added for ox_inv until I make a proper qbx bridge
-RegisterNetEvent('consumables:server:addThirst', addThirst)
 
-RegisterNetEvent('consumables:server:addHunger', addHunger)
+--- @deprecated confusing name
```

Comment:
I think the deprecation definition should direct the user to the new implementation, rather than the reason for deprecation

Reply:
Is currently good?